errors.go

```diff
@@ -101,20 +101,35 @@ func NewExitError(message interface{}, exitCode int) ExitCoder {
 // by overiding the ExitErrHandler function on an App or the package-global
 // OsExiter function.
 func Exit(message interface{}, exitCode int) ExitCoder {
+	var err error
+
+	switch e := message.(type) {
+	case ErrorFormatter:
```

Comment:
Is there any difference between this and the default handling?

Reply:
Well ErrorFormatter implies that the error message can be formatted dfferently than a "regular" error.